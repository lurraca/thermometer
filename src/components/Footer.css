footer {
    position:absolute;
    bottom:0;
    width:100%;
    height:60px;
    background:#6cf;
    text-align: center;
    vertical-align: middle;
    line-height: 60px;
}