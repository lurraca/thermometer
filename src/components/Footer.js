import React, { Component } from 'react';

import './Footer.css';

class Footer extends Component {

  render() {
    return(
      <footer>This is a test app</footer>
    )
  }
}

export default Footer;