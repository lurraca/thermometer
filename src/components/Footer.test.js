import React from 'react';
import { shallow } from 'enzyme';

import Footer from './Footer';

it('Footer renders correctly', () => {
  const renderedComponent = shallow(
    <Footer />
  );

  expect(renderedComponent).toMatchSnapshot();
});