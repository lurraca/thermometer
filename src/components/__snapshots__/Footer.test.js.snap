// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders correctly 1`] = `
ShallowWrapper {
  "complexSelector": ComplexSelector {
    "buildPredicate": [Function],
    "childrenOfNode": [Function],
    "findWhereUnwrapped": [Function],
  },
  "length": 1,
  "node": <footer>
    This is a test app
</footer>,
  "nodes": Array [
    <footer>
      This is a test app
</footer>,
  ],
  "options": Object {},
  "renderer": ReactShallowRenderer {
    "_instance":  {
      "_calledComponentWillUnmount": false,
      "_compositeType": 0,
      "_context": Object {},
      "_currentElement": <Footer />,
      "_debugID": 1,
      "_hostContainerInfo": null,
      "_hostParent": null,
      "_instance": Footer {
        "_reactInternalInstance": [Circular],
        "context": Object {},
        "props": Object {},
        "refs": Object {},
        "state": null,
        "updater": Object {
          "enqueueCallback": [Function],
          "enqueueCallbackInternal": [Function],
          "enqueueElementInternal": [Function],
          "enqueueForceUpdate": [Function],
          "enqueueReplaceState": [Function],
          "enqueueSetState": [Function],
          "isMounted": [Function],
          "validateCallback": [Function],
        },
      },
      "_mountOrder": 1,
      "_pendingCallbacks": null,
      "_pendingElement": null,
      "_pendingForceUpdate": false,
      "_pendingReplaceState": false,
      "_pendingStateQueue": null,
      "_renderedComponent": NoopInternalComponent {
        "_currentElement": <footer>
          This is a test app
</footer>,
        "_debugID": 2,
        "_renderedOutput": <footer>
          This is a test app
</footer>,
      },
      "_renderedNodeType": 0,
      "_rootNodeID": 0,
      "_topLevelWrapper": null,
      "_updateBatchNumber": null,
      "_warnedAboutRefsInRender": false,
    },
    "getRenderOutput": [Function],
    "render": [Function],
  },
  "root": [Circular],
  "unrendered": <Footer />,
}
`;
